games-rpg/devilutionx/patches/devilutionx-1.5.3.patchset

```diff
@@ -0,0 +1,23 @@
+From 520c9abcd5baca8d87efa6dfa66c37710000118a Mon Sep 17 00:00:00 2001
+From: Yourself <user@shredder.fritz.box>
+Date: Tue, 24 Sep 2024 15:49:31 +0200
+Subject: Specifying assets path for Haiku installs
+
+
+diff --git a/Source/utils/paths.cpp b/Source/utils/paths.cpp
+index e803436..dc65172 100644
+--- a/Source/utils/paths.cpp
++++ b/Source/utils/paths.cpp
+@@ -113,6 +113,9 @@ const std::string &ConfigPath()
+
+ const std::string &AssetsPath()
+ {
++#if defined(__HAIKU__)
++	assetsPath = "/boot/system/data/devilutionx/";
```

Comment:
This should use find_directory (see https://github.com/haikuports/haikuports/blob/c3d10adaa97ccb238a739915689e51d76a96c83c/app-emulation/x16-emulator/patches/x16_emulator-46.patchset#L29 for example)

Reply:
Done, I don't know how "good" the code is as I don't know C++ and just adapted what was done in the patch you linked me, but it works :)

Reply:
Looks good!